app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/pixels/DeviceShieldPixels.kt

```diff
@@ -749,6 +751,10 @@ class RealDeviceShieldPixels @Inject constructor(
         firePixel(DeviceShieldPixelNames.ATP_REPORT_VPN_NETWORK_STACK_CREATE_ERROR)
     }
 
+    override fun didOpenVpnOnboardingFromNotification(pixelName: String) {
+        firePixel(pixelName)
+    }
```

Comment:
I would rather define a pixel name under `DeviceShieldPixelNames` and then send the `pixelName` passed in the function call as a pixel parameter.

Do we need to add a daily pixel as well?

Reply:
The name of this pixel is composed from a pixel name and a suffix. I sent the whole thing instead of just the suffix as I think in the future we should move this code in one place.
I don't think we need a daily pixel here as this is sent when the app opens as a result of tapping on a notification and this is a one-off.